app/_assets/img/index.ts

```
@@ -0,0 +1,12 @@
+import githubTutorialStep2 from "./github_tutorial_step_2.webp";
+import githubTutorialStep3Gif from "./github_tutorial_step_3.gif";
+
+export const IMAGES = {
+    github: {
```

ESLint

[error] Delete `··`

(prettier/prettier)